pkg/virt-controller/watch/application.go

```diff
@@ -410,7 +417,24 @@ func (vca *VirtControllerApp) initCommon() {
 	vca.vmiController = NewVMIController(vca.templateService, vca.vmiInformer, vca.podInformer, vca.persistentVolumeClaimInformer, vca.vmiRecorder, vca.clientSet, vca.dataVolumeInformer)
 	recorder := vca.getNewRecorder(k8sv1.NamespaceAll, "node-controller")
 	vca.nodeController = NewNodeController(vca.clientSet, vca.nodeInformer, vca.vmiInformer, recorder)
-	vca.migrationController = NewMigrationController(vca.templateService, vca.vmiInformer, vca.podInformer, vca.migrationInformer, vca.vmiRecorder, vca.clientSet, vca.clusterConfig)
+
+	// Query the KubeVirt object to get Metrics config
+	kvList, err := vca.clientSet.KubeVirt(k8sv1.NamespaceAll).List(&v1.ListOptions{})
+	if err != nil {
+		panic(err)
```

Comment:
Looking at start of the method... I would use `golog.Fatal` for consistency... Maybe we can add little bit of context to this error, so the user knows what to do.